src/components/ctaBox/ctaBox.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import CTABox from './CTABox'
+
+storiesOf('CTA Box', module).add('default', () => (
+  <CTABox
+    title={'Hello'}
+    body={'this is the body'}
+    link={{ to: '/relative-url', text: 'This is a button' }}
```

Comment:
Could we make these editable (ie via `text` from `@storybook/addon-knobs`) so that we may edit the text from within Storybook? Makes it more dynamic and will be good once we get it published for Content/Designers. There should be a few examples of this in other stories!

Reply:
@ja9-look for the `link` prop you need to add the knobs for each property of the object passed

```
link={{
  to: text("Link URL", '/relative-url' ),
  text: text("Link Text", 'This is a button' )
}}
```

Reply:
thank you!!!

Reply:
Added text knob